TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -139,27 +151,43 @@ public class IncompatibleModsPanel : UIPanel {
         }
 
         private UIPanel CreateEntry(ref UIScrollablePanel parent, string name, ulong steamId) {
+            string caption = steamId == LOCAL_TMPE ? "Delete" : "Unsubscribe";
+
             UIPanel panel = parent.AddUIComponent<UIPanel>();
             panel.size = new Vector2(560, 50);
             panel.backgroundSprite = "ContentManagerItemBackground";
+
             UILabel label = panel.AddUIComponent<UILabel>();
             label.text = name;
             label.textAlignment = UIHorizontalAlignment.Left;
             label.relativePosition = new Vector2(10, 15);
-            CreateButton(panel, "Unsubscribe", (int) panel.width - 170, 10, delegate(UIComponent component, UIMouseEventParameter param) { UnsubscribeClick(component, param, steamId); });
+
+            CreateButton(panel, caption, (int)panel.width - 170, 10, delegate (UIComponent component, UIMouseEventParameter param) { UnsubscribeClick(component, param, steamId); });
+
             return panel;
         }
 
         private void UnsubscribeClick(UIComponent component, UIMouseEventParameter eventparam, ulong steamId) {
-            Log.Info("Trying to unsubscribe workshop item " + steamId);
+
+            bool success = false;
+
             component.isEnabled = false;
-            if (PlatformService.workshop.Unsubscribe(new PublishedFileId(steamId))) {
+
+            if (steamId == LOCAL_TMPE) {
+                Log.Info("Trying to delete local build of TM:PE");
+                success = DeleteLocalTMPE();
+            } else {
+                Log.Info("Trying to unsubscribe workshop item " + steamId);
+                success = PlatformService.workshop.Unsubscribe(new PublishedFileId(steamId));
+            }
+
+            if (success) {
```

Comment:
Would be great to figure out how to update list on every change (remove unsubscribed/deleted mods)

Reply:
Could you elaborate on this?

Reply:
Currently when you click unsubscribe/remove an action is performed but that mod entry is not removed after successful result iirc

Reply:
To keep it simple, why not just remove the mod entry panel when Unsubscribe/Delete is clicked?

Reasoning: If something goes wrong (eg. offline or some other error) there's not much the user can do about it, so rather than antagonise them, just ignore the error. It will affect very few users and next time they load the game they'll see the warning about the mod again - if it continues to fail they can just disable the mod checker or contact us.